apps/website/src/components/Playground/template/App.js

```diff
@@ -0,0 +1,41 @@
+import hljs from 'highlight.js/lib/core';
+import 'highlight.js/styles/vs.css';
+import beautify from 'js-beautify';
+import React from 'react';
+import css from 'highlight.js/lib/languages/css';
+import styles from './Styles';
+hljs.registerLanguage('css', css);
+
+export default function App() {
+  const [rules, setRules] = React.useState('');
+  const ref = React.useRef(null);
+  React.useEffect(() => {
+    const playgroundRenderer = {
+      id: 'playground',
+      insertCSSRules(cssRules) {
+        const raw = Object.values(cssRules).flat().join('\n');
+        const prettified = beautify.css_beautify(raw);
```

Comment:
```suggestion
        const prettified = beautify.css(raw, { indent_size: 2 });
```

nit: let's use "2" as indent size